lib/Sema/CSDiagnostics.cpp

```diff
@@ -835,6 +835,9 @@ bool MissingExplicitConversionFailure::diagnoseAsError() {
 
   auto fromType = getType(anchor)->getRValueType();
   Type toType = resolveType(ConvertingTo);
+  if (!toType->hasTypeRepr())
+    return false;
```

Comment:
I don't think this one's actually used yet (the ForceDowncast fix is only used in limited cases) but it's futureproofing.